const data = {
    apps: {
        recentlyAdded: [
            {
                id: "d6bda456-46d4-11ea-a4de-008cfa5ae621",
                system_id: "de",
                name: "Copy of Rstudio-DESeq2",
                description: "Rstudio DESeq2 VICE image",
                wiki_url: null,
                integration_date: null,
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2020-02-03T22:30:53.789Z",
                is_favorite: false,
                is_public: false,
            },
            {
                id: "a25da189-ffe6-4f61-9a91-704711f36bd7",
                system_id: "de",
                name: "New app",
                description: "asdf",
                wiki_url: null,
                integration_date: null,
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2014-12-08T06:01:37.242Z",
                is_favorite: false,
                is_public: false,
            },
            {
                id: "9ed50f34-fa8f-11e9-9bfc-008cfa5ae621",
                system_id: "de",
                name: "Copy of sequenceserver",
                description: "sequenceserver test app",
                wiki_url: null,
                integration_date: null,
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2019-10-29T21:04:26.403Z",
                is_favorite: false,
                is_public: false,
            },
            {
                id: "39896e30-3b5c-11e7-90ce-008cfa5ae621",
                system_id: "de",
                name: "Copy of New app",
                description: "asdf",
                wiki_url: null,
                integration_date: null,
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2017-05-17T23:54:54.624Z",
                is_favorite: false,
                is_public: false,
            },
            {
                id: "b390f5f0-4f13-11e9-8949-008cfa5ae621",
                system_id: "de",
                name: "test-vice-delay",
                description:
                    "A test app for testing a tool that has a significant delay or amount of processing to perform in its entrpoint.",
                wiki_url: null,
                integration_date: null,
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2019-03-25T15:36:04.700Z",
                is_favorite: false,
                is_public: false,
            },
            {
                id: "8bc52f58-4ff6-11e9-adb1-008cfa5ae621",
                system_id: "de",
                name: "wheee",
                description: "testing testing testing",
                wiki_url: null,
                integration_date: null,
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2019-03-26T18:39:53.693Z",
                is_favorite: false,
                is_public: false,
            },
            {
                id: "d3ec0a42-6142-11e9-b580-008cfa5ae621",
                system_id: "de",
                name: "test-vice-potree-entwine",
                description: "testing",
                wiki_url: null,
                integration_date: null,
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2019-04-17T18:58:46.260Z",
                is_favorite: false,
                is_public: false,
            },
            {
                id: "a74b9840-7b23-11e9-86f2-008cfa5ae621",
                system_id: "de",
                name: "Copy of GEA RNA-Seq QC Demo 0.1",
                description:
                    "This is a demonstration application for the genomics education alliance RNA-Seq pipeline",
                wiki_url: null,
                integration_date: null,
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2019-05-20T17:21:07.223Z",
                is_favorite: false,
                is_public: false,
            },
        ],
        public: [
            {
                id: "0dfe3e35-0000-4995-bc31-24b94ee7eada",
                system_id: "de",
                name: "QA_App-Endpoint-Test-Mod5_QA",
                description: "Testing new app endpoints.  Modified.",
                wiki_url: null,
                integration_date: "2020-03-16T07:14:46.656Z",
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2020-03-16T07:14:56.813Z",
                is_favorite: true,
                is_public: true,
            },
            {
                id: "c81731dc-6755-11ea-908b-c2a97b34bb42",
                system_id: "de",
                name: "QA_TestForMultiplePanelsTerrain_QA",
                description: "This is a new description for the shared app.",
                wiki_url: null,
                integration_date: "2020-03-16T07:14:35.636Z",
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2020-03-16T07:14:35.636Z",
                is_favorite: true,
                is_public: true,
            },
            {
                id: "9e989f50-6109-11ea-ab9d-008cfa5ae621",
                system_id: "de",
                name: "Snakemake-VICE",
                description:
                    "VICE app with jupyter lab and snakemake-minimal. Also conda and RNAseq analysis tools",
                wiki_url: null,
                integration_date: "2020-03-10T21:48:05.803Z",
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2020-03-10T21:48:05.803Z",
                is_favorite: false,
                is_public: true,
            },
            {
                id: "b91ffda4-4df0-11ea-bd40-008cfa5ae621",
                system_id: "de",
                name: "HTSeqQC 1.0",
                description:
                    "HTSeqQC is an automated quality control analysis tool for a single and paired-end high-throughput sequencing data (HTS) generated from Illumina sequencing platforms.",
                wiki_url: null,
                integration_date: "2020-03-09T21:13:54.884Z",
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2020-03-09T21:13:54.884Z",
                is_favorite: false,
                is_public: true,
            },
            {
                id: "aee316c6-15d5-11ea-933d-008cfa5ae621",
                system_id: "de",
                name: "Sequence Properties Analyser",
                description:
                    "Compares statistical properties of two sets of protein sequences.",
                wiki_url: null,
                integration_date: "2020-03-04T18:40:22.199Z",
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2020-03-04T18:40:22.199Z",
                is_favorite: false,
                is_public: true,
            },
            {
                id: "58f783bc-5c95-11ea-b10d-008cfa5ae621",
                system_id: "de",
                name: "ALLMAPS",
                description: "Performs ALLMAPS merge and path",
                wiki_url: null,
                integration_date: "2020-03-02T15:36:48.063Z",
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2020-03-02T15:36:48.063Z",
                is_favorite: false,
                is_public: true,
            },
            {
                id: "6ff82cda-5c98-11ea-aad3-008cfa5ae621",
                system_id: "de",
                name: "ALLMAPS merge",
                description:
                    "ALLMAPS orders scaffolds by maximizing colinearity to a collection of genetic/genomic maps into the final chromosome build.  This is step one: merging maps into a bed file and generating the weights file. http://goo.gl/KI61Ow",
                wiki_url: null,
                integration_date: "2020-03-02T15:33:49.287Z",
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2020-03-02T15:33:49.287Z",
                is_favorite: false,
                is_public: true,
            },
            {
                id: "a847402e-ff2a-11e9-815d-008cfa5ae621",
                system_id: "de",
                name: "QGIS Xpra Desktop",
                description:
                    "desktop running Xpra with QGIS, SAGA-GIS, GRASS, GDAL, PDAL",
                wiki_url: null,
                integration_date: "2020-02-17T22:20:28.291Z",
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2020-02-17T22:20:28.291Z",
                is_favorite: false,
                is_public: true,
            },
        ],
        popularFeatured: [
            {
                integration_date: "2021-08-06T00:20:46.090Z",
                description:
                    "Bourne Again Shell (BASH) terminal includes conda",
                is_favorite: false,
                name: "Cloud Shell",
                username: "tswetnam@iplantcollaborative.org",
                system_id: "de",
                is_public: true,
                id: "b29b7cf2-e4b1-11eb-8726-008cfa5ae621",
                edited_date: "2021-08-14T23:54:19.567Z",
                wiki_url: null,
                job_count: "17",
            },
            {
                integration_date: "2021-02-24T20:51:52.795Z",
                description: "Juypter Lab Data Science w/ Python, Julia, and R",
                is_favorite: false,
                name: "JupyterLab Datascience 2.2.9",
                username: "tswetnam@iplantcollaborative.org",
                system_id: "de",
                is_public: true,
                id: "07a2d5b2-76e2-11eb-be5f-008cfa5ae621",
                edited_date: "2021-06-10T13:34:55.443Z",
                wiki_url: null,
                job_count: "2",
            },
            {
                integration_date: "2021-06-25T21:33:17.760Z",
                description: "Xpra Ubuntu 20.04 with QGIS and OSGEO packages",
                is_favorite: false,
                name: "Xpra Ubuntu QGIS",
                username: "tswetnam@iplantcollaborative.org",
                system_id: "de",
                is_public: true,
                id: "9a133468-d5fc-11eb-9331-008cfa5ae621",
                edited_date: "2021-06-25T21:34:46.033Z",
                wiki_url: null,
                job_count: "1",
            },
        ],
    },

    analyses: {
        recent: [
            {
                description: "",
                name: "DE_Word_Count_analysis1",
                can_share: true,
                username: "ipcdev@iplantcollaborative.org",
                app_id: "67d15627-22c5-42bd-8daf-9af5deecceab",
                system_id: "de",
                app_disabled: false,
                batch: false,
                enddate: "1587023391999",
                status: "Submitted",
                id: "c1e6c4e8-7fb6-11ea-be0a-c2a97b34bb42",
                startdate: "1587023349739",
                app_description: "Counts the number of words in a file",
                wiki_url:
                    "https://wiki.cyverse.org/wiki/display/DEapps/Word+Count",
                notify: true,
                resultfolderid:
                    "/iplant/home/ipcdev/analyses_qa/DE_Word_Count_analysis1-2020-04-16-07-49-09.7",
                app_name: "DE Word Count",
            },
            {
                description: "",
                name: "DE_Word_Count_analysis1",
                can_share: true,
                username: "ipcdev@iplantcollaborative.org",
                app_id: "67d15627-22c5-42bd-8daf-9af5deecceab",
                system_id: "de",
                app_disabled: false,
                batch: false,
                enddate: "1587022622645",
                status: "Idle",
                id: "fbfa7d8e-7fb4-11ea-9730-c2a97b34bb42",
                startdate: "1587022588272",
                app_description: "Counts the number of words in a file",
                wiki_url:
                    "https://wiki.cyverse.org/wiki/display/DEapps/Word+Count",
                notify: true,
                resultfolderid:
                    "/iplant/home/ipcdev/analyses_qa/DE_Word_Count_analysis1-2020-04-16-07-36-28.2",
                app_name: "DE Word Count",
            },
            {
                description: "",
                name: "DE_Word_Count_analysis1",
                can_share: true,
                username: "ipcdev@iplantcollaborative.org",
                app_id: "67d15627-22c5-42bd-8daf-9af5deecceab",
                batch_status: {
                    total: 9,
                    completed: 9,
                    running: 0,
                    submitted: 0,
                },
                system_id: "de",
                app_disabled: false,
                batch: true,
                enddate: "1586950384244",
                status: "Completed",
                id: "a5e68c74-7f0c-11ea-be65-c2a97b34bb42",
                startdate: "1586950288785",
                app_description: "Counts the number of words in a file",
                notify: true,
                resultfolderid:
                    "/iplant/home/ipcdev/analyses_qa/DE_Word_Count_analysis1-2020-04-15-11-31-27.4",
                app_name: "DE Word Count",
            },
            {
                description: "",
                name: "ten-rules-jupyter_analysis1",
                can_share: true,
                username: "ipcdev@iplantcollaborative.org",
                app_id: "8ec235d8-f173-11e9-a56f-008cfa5ae621",
                system_id: "de",
                app_disabled: false,
                batch: false,
                enddate: "0",
                status: "Running",
                id: "8712ea7c-7f0c-11ea-be65-c2a97b34bb42",
                startdate: "1586950235151",
                app_description:
                    "Jupyter Notebooks for: Ten simple rules for writing and sharing computational analyses in Jupyter Notebooks. Rule A, Birmingham A, Zuniga C, Altintas I, Huang SC, Knight R, Moshiri N, Nguyen MH, Rosenthal SB, Pérez F, Rose PW. PLoS Comput Biol. 2019 Jul 25;15(7):e1007007. doi: 10.1371/journal.pcbi.1007007",
                interactive_urls: ["https://a444587f3.cyverse.run:4343"],
                notify: true,
                resultfolderid:
                    "/iplant/home/ipcdev/analyses_qa/ten-rules-jupyter_analysis1-2020-04-15-11-30-35.1",
                app_name: "ten-rules-jupyter",
            },
        ],
        running: [
            {
                description: "",
                name: "ten-rules-jupyter_analysis1",
                can_share: true,
                username: "ipcdev@iplantcollaborative.org",
                app_id: "8ec235d8-f173-11e9-a56f-008cfa5ae621",
                system_id: "de",
                app_disabled: false,
                batch: false,
                enddate: "0",
                status: "Running",
                id: "8712ea7c-7f0c-11ea-be65-c2a97b34bb42",
                startdate: "1586950235151",
                app_description:
                    "Jupyter Notebooks for: Ten simple rules for writing and sharing computational analyses in Jupyter Notebooks. Rule A, Birmingham A, Zuniga C, Altintas I, Huang SC, Knight R, Moshiri N, Nguyen MH, Rosenthal SB, Pérez F, Rose PW. PLoS Comput Biol. 2019 Jul 25;15(7):e1007007. doi: 10.1371/journal.pcbi.1007007",
                interactive_urls: ["https://a444587f3.cyverse.run:4343"],
                notify: true,
                resultfolderid:
                    "/iplant/home/ipcdev/analyses_qa/ten-rules-jupyter_analysis1-2020-04-15-11-30-35.1",
                app_name: "ten-rules-jupyter",
            },
        ],
    },

    instantLaunches: [
        {
            id: "1e2483cc-90ae-11eb-9bfd-62d47aced14b",
            added_by: "wregglej@iplantcollaborative.org",
            added_on: "2021-03-29T09:45:06.934586Z",
            quick_launch_id: "8ebc6c80-2058-4953-b4c9-ecf893deae03",
            quick_launch_name: "instant-launch-test",
            quick_launch_description: "",
            quick_launch_creator: "wregglej@iplantcollaborative.org",
            is_public: true,
            submission: {
                config: {
                    "d61e56f0-e921-11e9-8fe0-008cfa5ae621_d62788d8-e921-11e9-8fe0-008cfa5ae621":
                        [],
                },
                name: "jupyter-lab-instant-launch",
                app_id: "d61d9a26-e921-11e9-8fe0-008cfa5ae621",
                system_id: "de",
                debug: false,
                output_dir: "/iplant/home/wregglej/analyses_qa",
                notify: true,
            },
            app_id: "d61d9a26-e921-11e9-8fe0-008cfa5ae621",
            app_name: "JupyterLab-with-sql-1.0.9",
            app_description:
                "Jupyter Lab based on jupyter/datascience-notebook with updated iJab plugin, jupyterlab_irods and SQL ",
            app_deleted: false,
            app_disabled: false,
            integrator: "upendra_35@iplantcollaborative.org",
        },
    ],

    feeds: {
        news: [
            {
                id: "110 at https://cyverse.org",
                name: "Researchers Go on Offense, Defense in Battle Against Hackers",
                description:
                    "Researchers Go on Offense, Defense in Battle Against Hackers\nlittin\nMon, 12/09/2019 - 14:57",
                link: "https://cyverse.org/Researchers-Go-on-Offense-Defense-in-Battle-Against-Hackers",
                date_added: "2019-12-09T21:57:33.000Z",
            },
            {
                id: "88 at https://cyverse.org",
                name: "Researchers 'Light Up' the Plant Tree of Life",
                description:
                    "Researchers 'Light Up' the Plant Tree of Life\nlittin\nThu, 10/24/2019 - 08:42",
                link: "https://cyverse.org/Researchers-Light-Up-the-Plant-Tree-of-Life",
                date_added: "2019-10-24T15:42:11.000Z",
            },
            {
                id: "65 at https://cyverse.org",
                name: "CyVerse ‘Power User’ Joins Boyce Thompson Institute",
                description:
                    "CyVerse ‘Power User’ Joins Boyce Thompson Institute\nlittin\nThu, 10/03/2019 - 00:27",
                link: "https://cyverse.org/CyVerse-%E2%80%98Power-User%E2%80%99-Joins-Boyce-Thompson-Institute",
                date_added: "2019-10-03T07:27:35.000Z",
            },
            {
                id: "81 at https://cyverse.org",
                name: "ECSS Program Accelerates XSEDE User's Career in Geoinformatics",
                description:
                    "ECSS Program Accelerates XSEDE User's Career in Geoinformatics\nmwall\nMon, 10/07/2019 - 15:36",
                link: "https://cyverse.org/ECSS-Program-Accelerates-XSEDE-User%27s-Career-in-Geoinformatics",
                date_added: "2019-10-07T22:36:42.000Z",
            },
            {
                id: "5 at https://cyverse.org",
                name: "Introducing DataHog: A Novel New App to Analyze Stored Files",
                description:
                    "Introducing DataHog: A Novel New App to Analyze Stored Files\nlittin\nMon, 08/05/2019 - 22:53",
                link: "https://cyverse.org/Introducing%20DataHog%3A%20A%20Novel%20New%20App%20to%20Analyze%20Stored%20Files",
                date_added: "2019-08-06T05:53:13.000Z",
            },
            {
                id: "6 at https://cyverse.org",
                name: "Inaugural Open Science Course Teaches Best Practices",
                description:
                    "Inaugural Open Science Course Teaches Best Practices\nlittin\nMon, 08/05/2019 - 23:28",
                link: "https://cyverse.org/Inaugural-Open-Science-Course-Teaches-Best-Practices",
                date_added: "2019-08-06T06:28:31.000Z",
            },
            {
                id: "54 at https://cyverse.org",
                name: "NSF 2026 Idea Machine: Reinventing Scientific Talent",
                description:
                    "NSF 2026 Idea Machine: Reinventing Scientific Talent\nlittin\nMon, 09/02/2019 - 21:17",
                link: "https://cyverse.org/NSF-2026-Idea-Machine%3A-Reinventing-Scientific-Talent",
                date_added: "2019-09-03T04:17:03.000Z",
            },
            {
                id: "55 at https://cyverse.org",
                name: "UA Campus Comes Together to Celebrate Digital Literacy",
                description:
                    "UA Campus Comes Together to Celebrate Digital Literacy\nlittin\nMon, 09/02/2019 - 21:41",
                link: "https://cyverse.org/UA-Campus-Comes-Together-to-Celebrate-Digital-Literacy",
                date_added: "2019-09-03T04:41:02.000Z",
            },
        ],
        events: [
            {
                id: "129 at https://cyverse.org",
                name: "Webinar: Workflow Management Using Snakemake",
                description:
                    "Webinar: Workflow Management Using Snakemake\n\n              \n      tinal\nMon, 03/02/2020 - 13:46\n\n            Webinar: Workflow Management Using Snakemake\n\n      \n            March 27, 2020 | Virtual\n\n10am Pacific | 11am Mountain | 12noon Central | 1pm Eastern (DST)\n\n      \n            About this Webinar:\n\nFrom where and how to get data for your analysis, to where and how to treat the outputs, workflow managers can help you achieve better scientific reproducibility and scalability. In this webinar, University of Arizona graduate student Sateesh Peri will demonstrate Snakemake, an open source workflow management tool. Sateesh will first identify use cases where workflow managers are helpful for automation so that you can understand Snakefile componenets, such as the rules, inputs, outputs and actions. Then he will lead us through how to write and run a Snakefile in CyVerse VICE (Visual and Interactive Compute Environment). Once you learn to properly use Snakemake (or similar workflow management tools), keeping track of and sharing your work becomes second nature, not only saving you time whenever you need to re-run all or part of an analysis but helping you reduce errors that naturally get introduced whenever a non-automated activity is done (i.e., as part of the human condition of doing computational science and not being a bot!).\n\nRegister Here\n\nAbout the Presenter: \nSateesh PeriSateesh Peri is a PhD candidate in Eric Lyon’s lab at the University of Arizona and also works at CyVerse as a graduate intern. His primary research focus is on identification and characterization of long non-coding RNAs (lincRNAs); in his spare time, Sateesh likes learning and teaching computational skills to biologists (which is why we love working with him at CyVerse!). He is an open-science evangelist and an active instructor for Data Carpentry and for CyVerse's Foundational Open Source Skills (FOSS) and Container Camp.",
                link: "https://cyverse.org/webinar-workflow-management-using-snakemake",
                date_added: "2020-03-02T20:46:03.000Z",
            },
            {
                id: "124 at https://cyverse.org",
                name: "Webinar: Get Grounded in TERRA-REF: Publicly-accessible Hi-res Sensor Data for Crop Phenomics",
                description:
                    "Webinar: Get Grounded in TERRA-REF: Publicly-accessible Hi-res Sensor Data for Crop Phenomics\n\n              \n      tinal\nTue, 01/28/2020 - 13:47\n\n            Webinar: Get Grounded in TERRA-REF: Publicly-accessible, Hi-res Sensor Data for Crop Phenomics\n\n      \n            March 13, 2020 | Virtual\n\n10am Pacific | 11am Mountain | 12noon Central | 1pm Eastern (Daylight Savings Time)\n\n      \n            Presentation Slides\n\n      \n            Webinar Video-recording:\n\n\n\nAbout this Webinar: \n\n\n\n\n\n/*-->*/\n\n/*-->*/\n\n/*-->*/\n\n/*-->*/\n\nThis webinar will introduce the TERRA-REF project that produced a large, high resolution public domain dataset to advance high throughput plant phenomics. The TERRA REF dataset contains remote sensing, plant trait, environmental, agronomic and genomic data. The dataset was produced to enable researchers to evaluate and develop new computational pipelines for converting sensor data into biological and agronomic understanding and solutions. David LeBauer, who led development of TERRA-REF's data and computing pipeline will describe the project along with its computing infrastructure and datasets. He will showcase the diverse suite of data generated by thermal, hyperspectral, color, laser 3D, and active photosynthetic fluorescence cameras as well as hand measurements, environmental sensors, and deep genomic sequencing.\n\nAnyone interested in plant sciences, robotics, statistics, remote sensing, data science, and how to make such data FAIR (findable, accessible, interoperable, and reusable) will benefit from learning about TERRA-REF's publicly-funded and publicly-accessible crop phenotype datasets. David will describe how the TERRA REF data was produced and how users can learn more and get started using the data and contributing new algorithms to high throughput phenomics pipelines.\n\nAbout the Presenter:\n\n\n\nDavid LeBauer is the director of Data Science for the Agricultural Experiment Station at the University of Arizona. His research is focused on using science to engineer more sustainable and productive crops and agricultural systems. To support this effort, he develops open software and data to integrate data and knowledge across disciplines. Key projects include the Predictive Ecosystem Analyzer (PEcAn) framework for data synthesis and forecasting with crop and ecosystem models and the TERRA Reference phenotyping database and computing pipeline.",
                link: "https://cyverse.org/webinar-get-grounded-in-terra-ref-publicly-accessible-hi-res-sensor-data-for-crop-phenomics",
                date_added: "2020-01-28T20:47:52.000Z",
            },
            // fake a few more entries to stress the layout code a bit
            {
                id: "129 at https://cyverse.org",
                name: "Webinar: Workflow Management Using Snakemake",
                description:
                    "Webinar: Workflow Management Using Snakemake\n\n              \n      tinal\nMon, 03/02/2020 - 13:46\n\n            Webinar: Workflow Management Using Snakemake\n\n      \n            March 27, 2020 | Virtual\n\n10am Pacific | 11am Mountain | 12noon Central | 1pm Eastern (DST)\n\n      \n            About this Webinar:\n\nFrom where and how to get data for your analysis, to where and how to treat the outputs, workflow managers can help you achieve better scientific reproducibility and scalability. In this webinar, University of Arizona graduate student Sateesh Peri will demonstrate Snakemake, an open source workflow management tool. Sateesh will first identify use cases where workflow managers are helpful for automation so that you can understand Snakefile componenets, such as the rules, inputs, outputs and actions. Then he will lead us through how to write and run a Snakefile in CyVerse VICE (Visual and Interactive Compute Environment). Once you learn to properly use Snakemake (or similar workflow management tools), keeping track of and sharing your work becomes second nature, not only saving you time whenever you need to re-run all or part of an analysis but helping you reduce errors that naturally get introduced whenever a non-automated activity is done (i.e., as part of the human condition of doing computational science and not being a bot!).\n\nRegister Here\n\nAbout the Presenter: \nSateesh PeriSateesh Peri is a PhD candidate in Eric Lyon’s lab at the University of Arizona and also works at CyVerse as a graduate intern. His primary research focus is on identification and characterization of long non-coding RNAs (lincRNAs); in his spare time, Sateesh likes learning and teaching computational skills to biologists (which is why we love working with him at CyVerse!). He is an open-science evangelist and an active instructor for Data Carpentry and for CyVerse's Foundational Open Source Skills (FOSS) and Container Camp.",
                link: "https://cyverse.org/webinar-workflow-management-using-snakemake",
                date_added: "2020-03-02T20:46:03.000Z",
            },
            {
                id: "130 at https://cyverse.org",
                name: "Webinar: Get Grounded in TERRA-REF: Publicly-accessible Hi-res Sensor Data for Crop Phenomics",
                description:
                    "Webinar: Get Grounded in TERRA-REF: Publicly-accessible Hi-res Sensor Data for Crop Phenomics\n\n              \n      tinal\nTue, 01/28/2020 - 13:47\n\n            Webinar: Get Grounded in TERRA-REF: Publicly-accessible, Hi-res Sensor Data for Crop Phenomics\n\n      \n            March 13, 2020 | Virtual\n\n10am Pacific | 11am Mountain | 12noon Central | 1pm Eastern (Daylight Savings Time)\n\n      \n            Presentation Slides\n\n      \n            Webinar Video-recording:\n\n\n\nAbout this Webinar: \n\n\n\n\n\n/*-->*/\n\n/*-->*/\n\n/*-->*/\n\n/*-->*/\n\nThis webinar will introduce the TERRA-REF project that produced a large, high resolution public domain dataset to advance high throughput plant phenomics. The TERRA REF dataset contains remote sensing, plant trait, environmental, agronomic and genomic data. The dataset was produced to enable researchers to evaluate and develop new computational pipelines for converting sensor data into biological and agronomic understanding and solutions. David LeBauer, who led development of TERRA-REF's data and computing pipeline will describe the project along with its computing infrastructure and datasets. He will showcase the diverse suite of data generated by thermal, hyperspectral, color, laser 3D, and active photosynthetic fluorescence cameras as well as hand measurements, environmental sensors, and deep genomic sequencing.\n\nAnyone interested in plant sciences, robotics, statistics, remote sensing, data science, and how to make such data FAIR (findable, accessible, interoperable, and reusable) will benefit from learning about TERRA-REF's publicly-funded and publicly-accessible crop phenotype datasets. David will describe how the TERRA REF data was produced and how users can learn more and get started using the data and contributing new algorithms to high throughput phenomics pipelines.\n\nAbout the Presenter:\n\n\n\nDavid LeBauer is the director of Data Science for the Agricultural Experiment Station at the University of Arizona. His research is focused on using science to engineer more sustainable and productive crops and agricultural systems. To support this effort, he develops open software and data to integrate data and knowledge across disciplines. Key projects include the Predictive Ecosystem Analyzer (PEcAn) framework for data synthesis and forecasting with crop and ecosystem models and the TERRA Reference phenotyping database and computing pipeline.",
                link: "https://cyverse.org/webinar-get-grounded-in-terra-ref-publicly-accessible-hi-res-sensor-data-for-crop-phenomics",
                date_added: "2020-01-28T20:47:52.000Z",
            },
            {
                id: "131 at https://cyverse.org",
                name: "Webinar: Workflow Management Using Snakemake",
                description:
                    "Webinar: Workflow Management Using Snakemake\n\n              \n      tinal\nMon, 03/02/2020 - 13:46\n\n            Webinar: Workflow Management Using Snakemake\n\n      \n            March 27, 2020 | Virtual\n\n10am Pacific | 11am Mountain | 12noon Central | 1pm Eastern (DST)\n\n      \n            About this Webinar:\n\nFrom where and how to get data for your analysis, to where and how to treat the outputs, workflow managers can help you achieve better scientific reproducibility and scalability. In this webinar, University of Arizona graduate student Sateesh Peri will demonstrate Snakemake, an open source workflow management tool. Sateesh will first identify use cases where workflow managers are helpful for automation so that you can understand Snakefile componenets, such as the rules, inputs, outputs and actions. Then he will lead us through how to write and run a Snakefile in CyVerse VICE (Visual and Interactive Compute Environment). Once you learn to properly use Snakemake (or similar workflow management tools), keeping track of and sharing your work becomes second nature, not only saving you time whenever you need to re-run all or part of an analysis but helping you reduce errors that naturally get introduced whenever a non-automated activity is done (i.e., as part of the human condition of doing computational science and not being a bot!).\n\nRegister Here\n\nAbout the Presenter: \nSateesh PeriSateesh Peri is a PhD candidate in Eric Lyon’s lab at the University of Arizona and also works at CyVerse as a graduate intern. His primary research focus is on identification and characterization of long non-coding RNAs (lincRNAs); in his spare time, Sateesh likes learning and teaching computational skills to biologists (which is why we love working with him at CyVerse!). He is an open-science evangelist and an active instructor for Data Carpentry and for CyVerse's Foundational Open Source Skills (FOSS) and Container Camp.",
                link: "https://cyverse.org/webinar-workflow-management-using-snakemake",
                date_added: "2020-03-02T20:46:03.000Z",
            },
            {
                id: "132 at https://cyverse.org",
                name: "Webinar: Get Grounded in TERRA-REF: Publicly-accessible Hi-res Sensor Data for Crop Phenomics",
                description:
                    "Webinar: Get Grounded in TERRA-REF: Publicly-accessible Hi-res Sensor Data for Crop Phenomics\n\n              \n      tinal\nTue, 01/28/2020 - 13:47\n\n            Webinar: Get Grounded in TERRA-REF: Publicly-accessible, Hi-res Sensor Data for Crop Phenomics\n\n      \n            March 13, 2020 | Virtual\n\n10am Pacific | 11am Mountain | 12noon Central | 1pm Eastern (Daylight Savings Time)\n\n      \n            Presentation Slides\n\n      \n            Webinar Video-recording:\n\n\n\nAbout this Webinar: \n\n\n\n\n\n/*-->*/\n\n/*-->*/\n\n/*-->*/\n\n/*-->*/\n\nThis webinar will introduce the TERRA-REF project that produced a large, high resolution public domain dataset to advance high throughput plant phenomics. The TERRA REF dataset contains remote sensing, plant trait, environmental, agronomic and genomic data. The dataset was produced to enable researchers to evaluate and develop new computational pipelines for converting sensor data into biological and agronomic understanding and solutions. David LeBauer, who led development of TERRA-REF's data and computing pipeline will describe the project along with its computing infrastructure and datasets. He will showcase the diverse suite of data generated by thermal, hyperspectral, color, laser 3D, and active photosynthetic fluorescence cameras as well as hand measurements, environmental sensors, and deep genomic sequencing.\n\nAnyone interested in plant sciences, robotics, statistics, remote sensing, data science, and how to make such data FAIR (findable, accessible, interoperable, and reusable) will benefit from learning about TERRA-REF's publicly-funded and publicly-accessible crop phenotype datasets. David will describe how the TERRA REF data was produced and how users can learn more and get started using the data and contributing new algorithms to high throughput phenomics pipelines.\n\nAbout the Presenter:\n\n\n\nDavid LeBauer is the director of Data Science for the Agricultural Experiment Station at the University of Arizona. His research is focused on using science to engineer more sustainable and productive crops and agricultural systems. To support this effort, he develops open software and data to integrate data and knowledge across disciplines. Key projects include the Predictive Ecosystem Analyzer (PEcAn) framework for data synthesis and forecasting with crop and ecosystem models and the TERRA Reference phenotyping database and computing pipeline.",
                link: "https://cyverse.org/webinar-get-grounded-in-terra-ref-publicly-accessible-hi-res-sensor-data-for-crop-phenomics",
                date_added: "2020-01-28T20:47:52.000Z",
            },
        ],
        videos: [
            {
                id: "yt:video:Yf0w7UDRgpo",
                name: "Webinar: Identifying Transcription Factor Binding Sites from ChIP-seq Data",
                description:
                    "This webinar introduces the basic principles of analyzing chromatin immunoprecipitation sequencing (ChIP-seq) data, which is a powerful and versatile technique used for probing protein-DNA interactions. Thanks to the advances in sequencing technologies, high-throughput ChIP-seq offers researchers higher resolution, less noise, and greater coverage to identify and characterize DNA binding events at a genome-wide level compared to its array-based predecessor, ChIP-chip. In this webinar, Reetu covers the basics of ChIP-seq data handling and how to use public apps in the CyVerse Discovery Environment (DE) for processing and interpreting your ChIP-seq data. Reetu runs sample data with the CyVerse apps to demonstrate basic but key steps of ChIP-seq analysis: quality control of data, mapping data to a reference genome and peak calling. This webinar helps participants efficiently manage and analyze ChIP-seq data, starting from sequence reads through peak calling.\n\nWebinar materials, including interactive slides, are available here: https://cyverse.org/webinar-identifying-transcription-factor-binding-sites-from-ChIP-seq-data\n\nCyVerse provides powerful infrastructure for computational research involving large datasets and complex analyses. Using CyVerse, teams can overcome scaling and complexity challenges to tackle questions that previously were unapproachable.\n\n**Get Started!**\nCreate your Account: http://cyver.se/account \nLearn about CyVerse: https://cyverse.org \nLearn how to use CyVerse: http://cyver.se/learn \n\n**Make CyVerse work for you!**\nUse hundreds of bioinformatics apps and manage data in a simple web interface\nLaunch the Discovery Environment: http://cyver.se/de \n\n**Create a custom cloud-based scientific analysis platform or use a ready-made one for your area of scientific interest.**\nLaunch Atmosphere: http://cyver.se/atmo \n\n**Securely store data for active analyses or sharing with your collaborators.**\nLaunch the Data Store: http://cyver.se/datastore \n\n**Access discoverable and reusable datasets with metadata features and functions.**\nLaunch the Data Commons: http://cyver.se/datacommons\n\n**Exchange, explore, and analyze biological images and their metadata.**\nLaunch BisQue: http://cyver.se/bisque \n\n**Follow us!**\nTwitter: http://cyver.se/twitter \nFacebook: http://cyver.se/Facebook \nLinkedIn: http://cyver.se/LinkedIn \nYouTube: http://cyver.se/YouTube \nGitHub: http://cyver.se/GitHub \n\nStill have questions?\nEmail info@cyverse.org\n\nThis material is based upon work supported by the National Science Foundation under Grant Nos. DBI-0735191, DBI-1265383, and DBI-1743442.",
                date_added: "2020-04-24T19:01:32.000Z",
                author: "CyVerse.org",
                publication_date: "2020-04-24T19:01:32.000Z",
                link: "https://www.youtube.com/watch?v=Yf0w7UDRgpo",
                thumbnailUrl:
                    "https://i2.ytimg.com/vi/Yf0w7UDRgpo/hqdefault.jpg",
            },
            {
                id: "yt:video:s9DOBPlDZ3U",
                name: "Webinar: Get a Taste of Orange: An Intro to Data Mining and Machine Learning",
                description:
                    "Need something a little refreshing while you're hunkered down during this challenging time? How about a taste of Orange? In this finale to our spring webinars, researchers and students interested in approaching machine learning and data mining to conduct basic data analysis get a \"fruitful and fun\" introduction to Orange, an open-source machine learning and data visualization toolbox for novice and expert alike. Presenter Dr. Francesca Vitali of the University of Arizona's Center for Innovation in Brain Science (CIBS) teaches us how to use Orange to construct basic data analysis workflows. Using the interactive toolbox, she demonstrates key features of Orange and provides instructions and ideas on how to use Orange to conduct preliminary data analysis and basic machine learning to facilitate hypothesis generation from your data.\n\nCyVerse provides powerful infrastructure for computational research involving large datasets and complex analyses. Using CyVerse, teams can overcome scaling and complexity challenges to tackle questions that previously were unapproachable.\n\n**Get Started!**\nCreate your Account: http://cyver.se/account \nLearn about CyVerse: https://cyverse.org \nLearn how to use CyVerse: http://cyver.se/learn \n\n**Make CyVerse work for you!**\nUse hundreds of bioinformatics apps and manage data in a simple web interface\nLaunch the Discovery Environment: http://cyver.se/de \n\n**Create a custom cloud-based scientific analysis platform or use a ready-made one for your area of scientific interest.**\nLaunch Atmosphere: http://cyver.se/atmo \n\n**Securely store data for active analyses or sharing with your collaborators.**\nLaunch the Data Store: http://cyver.se/datastore \n\n**Access discoverable and reusable datasets with metadata features and functions.**\nLaunch the Data Commons: http://cyver.se/datacommons\n\n**Exchange, explore, and analyze biological images and their metadata.**\nLaunch BisQue: http://cyver.se/bisque \n\n**Follow us!**\nTwitter: http://cyver.se/twitter \nFacebook: http://cyver.se/Facebook \nLinkedIn: http://cyver.se/LinkedIn \nYouTube: http://cyver.se/YouTube \nGitHub: http://cyver.se/GitHub \n\nStill have questions?\nEmail info@cyverse.org\n\nThis material is based upon work supported by the National Science Foundation under Grant Nos. DBI-0735191, DBI-1265383, and DBI-1743442.",
                date_added: "2020-05-08T20:05:11.000Z",
                author: "CyVerse.org",
                publication_date: "2020-05-08T20:05:11.000Z",
                link: "https://www.youtube.com/watch?v=s9DOBPlDZ3U",
                thumbnailUrl:
                    "https://i4.ytimg.com/vi/s9DOBPlDZ3U/hqdefault.jpg",
            },
            {
                id: "yt:video:k9lKxp3ITY0",
                name: "DNA Subway Short Tour",
                description:
                    "DNA Subway is an educational bioinformatics platform developed by CyVerse. It bundles research-grade bioinformatics tools, high-performance computing, and databases into workflows with an easy-to-use interface. “Riding” DNA Subway lines, students can predict and annotate genes in up to 150kb of DNA (Red Line), identify homologs in sequenced genomes (Yellow Line), identify species using DNA barcodes and phylogenetic trees (Blue Line), examine RNA-Seq datasets for differential transcript abundance (Green Line), and analyze metabarcoding and eDNA samples using QIIME (Purple Line).\n\n\nTry DNA Subway at: https://dnasubway.cyverse.org/\nSee documentation at: https://cyverse-dnasubway-guide.readthedocs-hosted.com/en/latest/",
                date_added: "2020-05-20T17:45:09.000Z",
                author: "CyVerse.org",
                publication_date: "2020-05-20T17:45:09.000Z",
                link: "https://www.youtube.com/watch?v=k9lKxp3ITY0",
                thumbnailUrl:
                    "https://i4.ytimg.com/vi/k9lKxp3ITY0/hqdefault.jpg",
            },
            {
                id: "yt:video:DqqWdejDtDA",
                name: "CyVerse Foundational Open Science Skills, BOSC2020",
                description:
                    "CyVerse's Tyson Swetnam's presentation for the Bioinformatics Open Source Community (BOSC) 2020 conference on Foundational Open Science Skills (FOSS) training and how you can gain and apply these critical computational research skills through CyVerse Learning resources--available now online!",
                date_added: "2020-07-07T16:49:08.000Z",
                author: "CyVerse.org",
                publication_date: "2020-07-07T16:49:08.000Z",
                link: "https://www.youtube.com/watch?v=DqqWdejDtDA",
                thumbnailUrl:
                    "https://i1.ytimg.com/vi/DqqWdejDtDA/hqdefault.jpg",
            },
            {
                id: "yt:video:GlTaC4Sag4Y",
                name: "Webinar: Hit the Ground Running with SNP Calling for PopGen and Evolutionary Analyses",
                description:
                    "Presenter: Jacob Landis (https://jacoblandis.com/), Cornell University \n\nThe fields of population genetics (PopGen) and species evolution have especially benefitted from myriad advances in technology and computational methods of the 4th Industrial Revolution. In this kickoff to our Fall webinars, Jacob Landis of Cornell University's School of Integrative Plant Science will help you 'hit the ground running' for analyzing single nucleotide polymorphisms (SNPs). He'll cover the basics of different SNP calling techniques, such as RADSeq, HybSeq, and genome resequencing and then lead a quick tutorial for GATK, a frequently-used software package developed at the Broad Institute to analyze next-generation sequencing data. As a user-requested topic, this webinar is aimed at scientists who want to call SNPs for either PopGen or evolutionary analyses but just don't know where to begin. Join us to learn about some of the different techniques you can use to get a jumpstart on your SNP analyses.\n\nWebinar materials, including slides and description, can be found here: https://cyverse.org/webinar_SNP_analysis\n\n**Get Started!**\nCreate your Account: http://cyver.se/account\nLearn about CyVerse: https://cyverse.org\nLearn how to use CyVerse: http://cyver.se/learn\n\n**Make CyVerse work for you!**\nUse hundreds of bioinformatics apps and manage data in a simple web interface\nLaunch the Discovery Environment: http://cyver.se/de\n\nCreate a custom cloud-based scientific analysis platform or use a ready-made one for your area of scientific interest.\nLaunch Atmosphere: http://cyver.se/atmo\n\nSecurely store data for active analyses or sharing with your collaborators.\nLaunch the Data Store: http://cyver.se/datastore\n\nAccess discoverable and reusable data with metadata features and functions.\nLaunch the Data Commons: http://cyver.se/datacommons\n\nExchange, explore, and analyze biological images and their metadata.\nLaunch BisQue: http://cyver.se/bisque\n\n**Be sure to follow us!**\nTwitter: http://cyver.se/twitter\nFacebook: http://cyver.se/Facebook\nLinkedIn: http://cyver.se/LinkedIn\nYouTube: http://cyver.se/YouTube\nGitHub: http://cyver.se/GitHub\n\nStill have questions?\nEmail info@cyverse.org\n\nThis material is based upon work supported by the National Science Foundation under Grant Nos. DBI-0735191, DBI-1265383, DBI-1743442",
                date_added: "2020-09-11T21:47:26.000Z",
                author: "CyVerse.org",
                publication_date: "2020-09-11T21:47:26.000Z",
                link: "https://www.youtube.com/watch?v=GlTaC4Sag4Y",
                thumbnailUrl:
                    "https://i4.ytimg.com/vi/GlTaC4Sag4Y/hqdefault.jpg",
            },
            {
                id: "yt:video:vnOmfNaW5Vw",
                name: "Webinar: Using iCommands for Flexibility and Control in Moving Data in CyVerse",
                description:
                    "With capabilities to manage all your data throughout the research cycle and 100 GB of available data storage with your account, keeping all your data in CyVerse makes sense.\n\nHowever, moving large amounts of data can be daunting. For flexible and easy control of your data transfers in and out of CyVerse, you can use iCommands, a collection of command line tools developed by iRODS, the technology underlying the Data Store. In this webinar, CyVerse Co-PI Eric Lyons will show you some tips and tricks for the most reliable data transfers, starting with reviewing some Unix basics and how to use iCommands for tasks such as up/downloading from your local computer to the Data Store and managing directories. Attendees will need to have access to a Linux command line prompt and have installed iCommands, but even if you're not yet adept at the command line, understanding proper data management basics through this webinar is a good start.\n\n\n**Get Started!**\nCreate your Account: http://cyver.se/account \nLearn about CyVerse: https://cyverse.org \nLearn how to use CyVerse: http://cyver.se/learn \n\n**Make CyVerse work for you!**\nUse hundreds of bioinformatics apps and manage data in a simple web interface\nLaunch the Discovery Environment: http://cyver.se/de \n\n**Securely store data for active analyses or sharing with your collaborators.**\nLaunch the Data Store: http://cyver.se/datastore \n\n**Access discoverable and reusable datasets with metadata features and functions.**\nLaunch the Data Commons: http://cyver.se/datacommons\n\n**Exchange, explore, and analyze biological images and their metadata.**\nLaunch BisQue: http://cyver.se/bisque \n\n**Follow us!**\nTwitter: http://cyver.se/twitter \nFacebook: http://cyver.se/Facebook \nLinkedIn: http://cyver.se/LinkedIn \nYouTube: http://cyver.se/YouTube \nGitHub: http://cyver.se/GitHub \n\nStill have questions?\nEmail info@cyverse.org\n\nThis material is based upon work supported by the National Science Foundation under Grant Nos. DBI-0735191, DBI-1265383, and DBI-1743442.",
                date_added: "2020-09-25T20:45:10.000Z",
                author: "CyVerse.org",
                publication_date: "2020-09-25T20:45:10.000Z",
                link: "https://www.youtube.com/watch?v=vnOmfNaW5Vw",
                thumbnailUrl:
                    "https://i3.ytimg.com/vi/vnOmfNaW5Vw/hqdefault.jpg",
            },
            {
                id: "yt:video:jLTcKTa8RKY",
                name: "Webinar: CyVerse's Terrain API for Programmatic Access to Backend Services",
                description:
                    "CyVerse Software Engineer Sarah Roberts takes you through a Jupyter Notebook to apply the CyVerse Terrain API (Application Programming Interface) to access backend services that help you integrate CyVerse resources into your web applications and analyses.\n\n**Get Started!**\nCreate your Account: http://cyver.se/account\nLearn about CyVerse: https://cyverse.org\nLearn how to use CyVerse: http://cyver.se/learn\n\n**Make CyVerse work for you!**\nUse hundreds of bioinformatics apps and manage data in a simple web interface\nLaunch the Discovery Environment: http://cyver.se/de\n\nCreate a custom cloud-based scientific analysis platform or use a ready-made one for your area of scientific interest.\nLaunch Atmosphere: http://cyver.se/atmo\n\nSecurely store data for active analyses or sharing with your collaborators.\nLaunch the Data Store: http://cyver.se/datastore\n\nAccess discoverable and reusable data with metadata features and functions.\nLaunch the Data Commons: http://cyver.se/datacommons\n\nExchange, explore, and analyze biological images and their metadata.\nLaunch BisQue: http://cyver.se/bisque\n\n**Be sure to follow us!**\nTwitter: http://cyver.se/twitter\nFacebook: http://cyver.se/Facebook\nLinkedIn: http://cyver.se/LinkedIn\nYouTube: http://cyver.se/YouTube\nGitHub: http://cyver.se/GitHub\n\nStill have questions?\nEmail info@cyverse.org\n\nThis material is based upon work supported by the National Science Foundation under Grant Nos. DBI-0735191, DBI-1265383, DBI-1743442",
                date_added: "2020-10-11T07:00:12.000Z",
                author: "CyVerse.org",
                publication_date: "2020-10-11T07:00:12.000Z",
                link: "https://www.youtube.com/watch?v=jLTcKTa8RKY",
                thumbnailUrl:
                    "https://i3.ytimg.com/vi/jLTcKTa8RKY/hqdefault.jpg",
            },
            {
                id: "yt:video:zCksDE5e2Lc",
                name: "Webinar: Advanced Capabilities Orange",
                description:
                    "In this webinar, Dr Francesca Vitali uses a Docker container in CyVerse to demonstrate some advanced capabilities of the Orange software program.\n\n**Get Started!**\nCreate your Account: http://cyver.se/account\nLearn about CyVerse: https://cyverse.org\nLearn how to use CyVerse: http://cyver.se/learn\n\n**Make CyVerse work for you!**\nUse hundreds of bioinformatics apps and manage data in a simple web interface\nLaunch the Discovery Environment: http://cyver.se/de\n\nCreate a custom cloud-based scientific analysis platform or use a ready-made one for your area of scientific interest.\nLaunch Atmosphere: http://cyver.se/atmo\n\nSecurely store data for active analyses or sharing with your collaborators.\nLaunch the Data Store: http://cyver.se/datastore\n\nAccess discoverable and reusable data with metadata features and functions.\nLaunch the Data Commons: http://cyver.se/datacommons\n\nExchange, explore, and analyze biological images and their metadata.\nLaunch BisQue: http://cyver.se/bisque\n\n**Be sure to follow us!**\nTwitter: http://cyver.se/twitter\nFacebook: http://cyver.se/Facebook\nLinkedIn: http://cyver.se/LinkedIn\nYouTube: http://cyver.se/YouTube\nGitHub: http://cyver.se/GitHub\n\nStill have questions?\nEmail info@cyverse.org\n\nThis material is based upon work supported by the National Science Foundation under Grant Nos. DBI-0735191, DBI-1265383, DBI-1743442",
                date_added: "2020-10-24T17:11:34.000Z",
                author: "CyVerse.org",
                publication_date: "2020-10-24T17:11:34.000Z",
                link: "https://www.youtube.com/watch?v=zCksDE5e2Lc",
                thumbnailUrl:
                    "https://i3.ytimg.com/vi/zCksDE5e2Lc/hqdefault.jpg",
            },
            {
                id: "yt:video:zBCB2JYkFtk",
                name: "Webinar: A Preview of CyVerse's New User Interface for the Discovery Environment",
                description:
                    'After 10+ years of service, the CyVerse Discovery Environment has a new User Interface!  Software Dev Team Lead, Sriram Srinivasan, previews our new User Interface ("Sonora") including all kinds of new functionality (e.g., use "Bags" to collect resources for sharing with one click!), while maintaining a strong foundation of easy access to your Data, Apps, and Analyses.\n\n**Get Started!**\nCreate your Account: http://cyver.se/account\nLearn about CyVerse: https://cyverse.org\nLearn how to use CyVerse: http://cyver.se/learn\n\n**Make CyVerse work for you!**\nUse hundreds of bioinformatics apps and manage data in a simple web interface\nLaunch the Discovery Environment: http://cyver.se/de\n\nCreate a custom cloud-based scientific analysis platform or use a ready-made one for your area of scientific interest.\nLaunch Atmosphere: http://cyver.se/atmo\n\nSecurely store data for active analyses or sharing with your collaborators.\nLaunch the Data Store: http://cyver.se/datastore\n\nAccess discoverable and reusable data with metadata features and functions.\nLaunch the Data Commons: http://cyver.se/datacommons\n\nExchange, explore, and analyze biological images and their metadata.\nLaunch BisQue: http://cyver.se/bisque\n\n**Be sure to follow us!**\nTwitter: http://cyver.se/twitter\nFacebook: http://cyver.se/Facebook\nLinkedIn: http://cyver.se/LinkedIn\nYouTube: http://cyver.se/YouTube\nGitHub: http://cyver.se/GitHub\n\nStill have questions?\nEmail info@cyverse.org\n\nThis material is based upon work supported by the National Science Foundation under Grant Nos. DBI-0735191, DBI-1265383, DBI-1743442',
                date_added: "2020-11-07T17:15:51.000Z",
                author: "CyVerse.org",
                publication_date: "2020-11-07T17:15:51.000Z",
                link: "https://www.youtube.com/watch?v=zBCB2JYkFtk",
                thumbnailUrl:
                    "https://i3.ytimg.com/vi/zBCB2JYkFtk/hqdefault.jpg",
            },
            {
                id: "yt:video:N-p5tkGgE0U",
                name: "Webinar: PhytoOracle - A Workflow Manager for Phenomic Data  Processing",
                description:
                    "Using phenomic data generated by the the UA's Gantry, the world's largest outdoor phenotyping platform, four student developers present their work on creating the scalable, distributed workflow manager, PhytoOracle, for analyzing high-throughput phenotyping data. Michele Cosi, Ariyan Zarei, Emmanuel Gonzalez, and Travis Simmons discuss how they collaborated to leverage CyVerse, container technology, and open-source tools (e.g., CC-Tools) to create PhytoOracle and how you can adapt their containerized workflow manager for HT phenotyping on your image data.\n\n**Get Started!**\nCreate your Account: http://cyver.se/account\nLearn about CyVerse: https://cyverse.org\nLearn how to use CyVerse: http://cyver.se/learn\n\n**Make CyVerse work for you!**\nUse hundreds of bioinformatics apps and manage data in a simple web interface\nLaunch the Discovery Environment: http://cyver.se/de\n\nCreate a custom cloud-based scientific analysis platform or use a ready-made one for your area of scientific interest.\nLaunch Atmosphere: http://cyver.se/atmo\n\nSecurely store data for active analyses or sharing with your collaborators.\nLaunch the Data Store: http://cyver.se/datastore\n\nAccess discoverable and reusable data with metadata features and functions.\nLaunch the Data Commons: http://cyver.se/datacommons\n\nExchange, explore, and analyze biological images and their metadata.\nLaunch BisQue: http://cyver.se/bisque\n\n**Be sure to follow us!**\nTwitter: http://cyver.se/twitter\nFacebook: http://cyver.se/Facebook\nLinkedIn: http://cyver.se/LinkedIn\nYouTube: http://cyver.se/YouTube\nGitHub: http://cyver.se/GitHub\n\nStill have questions?\nEmail info@cyverse.org\n\nThis material is based upon work supported by the National Science Foundation under Grant Nos. DBI-0735191, DBI-1265383, DBI-1743442",
                date_added: "2020-11-23T14:50:13.000Z",
                author: "CyVerse.org",
                publication_date: "2020-11-23T14:50:13.000Z",
                link: "https://www.youtube.com/watch?v=N-p5tkGgE0U",
                thumbnailUrl:
                    "https://i3.ytimg.com/vi/N-p5tkGgE0U/hqdefault.jpg",
            },
        ],
    },
};

export default data;
